platform/cosmos/client.go

```diff
@@ -105,15 +89,6 @@ func (c *Client) GetUnbondingDelegations(address string) (delegations []Unbondin
 
 func (c *Client) GetAccount(address string) (result Account, err error) {
 	path := fmt.Sprintf("auth/accounts/%s", address)
-	return result, c.Get(&result, path, nil)
-}
-
-var everstakeValidator = Validator{
```

Comment:
Are you preserving this validator?

Reply:
now this is returned inside the validator endpoint